content/blog/2020/06/camel-k-release-1.0.0/index.md

```diff
@@ -0,0 +1,250 @@
+you can read some introductory blog posts here ([1 - introducing camel k ](https://nicolaferraro.me/2018/10/15/introducing-camel-k/), [2 - camel k on knative](https://www.nicolaferraro.me/2018/12/10/camel-k-on-knative/))
+or look at the Apache Camel website that contains a [Camel K section](/camel-k/latest/)
+with a lot of material that is automatically generated from the [Github repository](https://github.com/apache/camel-k).
+
+## User experience
+
+Camel K development style is minimalistic: you need just to write a single file with your integration routes and you can immediately 
+run them on any Kubernetes cluster. This way of defining things is common to many FaaS platforms (although Camel K is not a proper FaaS platform, 
+but a lightweight *integration* platform) and it's technically difficult to provide IDE support, such as code completion and other utilities, 
+to developers.
+
+But now we've it. The Red Hat Integration tooling team has created some cool extensions for VS Code that make the development experience 
+with Camel K even more exciting.
+You don't need to remember the Camel DSL syntax, the IDE will give you suggestions and error highlighting.
+
+![IDE Autocompletion](./ide-autocompletion.gif)
+
+Code completion works with Java code, but it's not only limited to it: you also have suggestions and documentation out of the box when writing the Camel URIs and property files.
+And you also have many options to run integrations and interact with them, all integrated in the IDE.
+
+Just install the VS Code [Extension Pack for Apache Camel by Red Hat](https://marketplace.visualstudio.com/items?itemName=redhat.apache-camel-extension-pack) to have all these features available.
+
+
+## Getting started tutorials
+
+Good tools are fundamental to have a great development experience with Camel K, but then you need to learn what you can do with such a great power.
+We've created a new repository in the Apache organization that hosts getting started examples: the [camel-k-examples](https://github.com/apache/camel-k-examples) repository.
+
+So far we've added guides that drive you through:
+
+- [01 Basic](https://github.com/apache/camel-k-examples/tree/master/01-basic): Learn the basics of Camel K and some interesting use cases
+- [02 Serverless APIs](https://github.com/apache/camel-k-examples/tree/master/02-serverless-api): How to design a serverless (i.e. auto-scaling, scaling to zero) API and run it in a few minutes
+
+The **basic quickstart is [also available online](https://learn.openshift.com/middleware/courses/middleware-camelk/camel-k-basic)**, so you can have a look at how camel k works **without installing anything on your laptop**.
+
+More examples are expected to come in the following months. You are also welcome if you want to help us by **contributing your own**.
+They are based on the [VSCode Didact](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-didact) project, that provides an
+awesome user experience.
+
```

Comment:
We could also add a section about the misc examples we have here
https://github.com/apache/camel-k/tree/master/examples

I do like the tutorials but they kinda assume you have the time and want to use Visual Code.

For people that just want to hack from CLI and a plain text editor (notepad++, emacs etc) then I would like to point them to these standalone. We could in the future port them or some of them to that example git repo.

There are some good ones here we could point to at first:

kamel run Sample.java
kamel run hello.xml
kamel run simple.js